pkgs/applications/science/logic/hol/default.nix

```diff
@@ -46,8 +46,8 @@ stdenv.mkDerivation {
     cd ${holsubdir}
 
     substituteInPlace tools/Holmake/Holmake_types.sml \
-      --replace "\"/bin/mv\"" "\"mv\"" \
-      --replace "\"/bin/cp\"" "\"cp\""
+      --replace "\"/bin/" "\"" \
```

Comment:
This could lead that binaries are used which are not available for this package. How many different program need patching?

Reply:
> This could lead that binaries are used which are not available for this package.

Only stuff from coreutils and some binaries from itself. @potan pls check whether binaries work ^^

Reply:
This code from Holmake test works correct.

all:
	$(CP) Holmakefile output